web/src/lib/components/album-page/album-description.svelte

```diff
@@ -0,0 +1,44 @@
+<script lang="ts">
+  import { autoGrowHeight } from '$lib/utils/autogrow';
+  import { updateAlbumInfo } from '@immich/sdk';
+  import { handleError } from '$lib/utils/handle-error';
+
+  export let id: string;
+  export let description: string;
+  export let isOwned: boolean;
+
+  $: newDescription = description;
+
+  const handleUpdateDescription = async () => {
+    if (newDescription === description) {
+      return;
+    }
+
+    try {
+      await updateAlbumInfo({
+        id,
+        updateAlbumDto: {
+          description: newDescription,
+        },
+      });
+      description = newDescription;
+    } catch (error) {
+      handleError(error, 'Error updating album description');
+    }
+  };
```

Comment:
I think you missed adding the notification thing back in, right? You only deleted it but didn't move it below the try/catch